### DISCUSSION TITLE
HLS (m3u8) live streaming

### DISCUSSION THREAD

**Discussion by zadorozhko** [alexballas/go2tv#91] [Show and tell]
Hi guys,
I'm looked for HLS support in go2tv, it's a main streaming protocol for Apple devices, but found this feature [was already rejected](https://github.com/alexballas/go2tv/issues/84#issuecomment-2156221223). Alex said no one need this, no devices are supporting m3u8 as input, but my $5 EZCast WiFi->HDMI dongle do. So, for those who need this, I want to share a tiny command-line player written in node.js - [dlnacast](https://github.com/zadorozhko/dlnacast/). It can discover devices, cast to them by name, not only by http://ip:port/xml (useful when target is on DHCP), and use local/network files as well as http://.m3u8 streams. It have no gui and no on-the-fly transcoding.

**Comment by alexballas** [comment]
Go2TV supports m3u8 to chromecast now :)
